lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/ConfigurationManager.cs

```diff
@@ -152,6 +154,16 @@ internal sealed class ConfigurationManager
             {
                 Exceptions.Throw(e, "Unable to deserialize map input pipeline data converter configuration", Logger);
             }
+
+            try
+            {
+                _checkpointConfiguration =
+                    configurationSerializer.FromString(checkpointConfiguration);
+            }
+            catch (Exception e)
+            {
+                Exceptions.Throw(e, "Unable to deserialize checkpoint configuration", Logger);
```

Comment:
Remove the `Exceptions` use while you are at it :). Better yet, remove the useless `catch` here altogether.

Reply:
Just to follow the existing pattern. Yes, the try catch is not necessary.